index.html

```diff
@@ -8,7 +8,7 @@
   <title>Camilo Vela Portfolio</title>
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;700&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;700&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
 </head>
 <body>

```

Comment:
There is a huge gap consistently on the right side of the screen. It might be because you have used a fixed width for the whole body beyond 768px. Consider fixing it.
![image](https://user-images.githubusercontent.com/33268927/177908022-e652c87f-01be-47b9-ad70-43b16a0516c4.png)

Reply:
Done ✅ 

I've done smooth changes to the webpage desktop display (compared with the Figma template) to solve this issue and improve UX. 🖥️ 

These changes were using a combination of `width: 100%` and `margin: auto` for lateral margins. The choice of which change to use in any element was made using just own criteria.

Let me know if it is OK or which kind of display you suggest. 😄